eZ/Publish/API/Repository/Tests/SearchServiceLocationTest.php

```diff
@@ -101,6 +101,35 @@ protected function createMultipleCountriesContent()
         return $content;
     }
 
+    /**
+     * Create test Content with non-printable characters in name.
+     *
+     * @return \eZ\Publish\API\Repository\Values\Content\Content
+     */
+    protected function createFolderWithNonPrintableUtf8Characters()
```

Comment:
CS:
For `7.5` (because it's current base) it should be:
```suggestion
    protected function createFolderWithNonPrintableUtf8Characters(): Content
```

For `6.13`:
```suggestion
    /**
     * @return \eZ\Publish\API\Repository\Values\Content\Content
     */
    protected function createFolderWithNonPrintableUtf8Characters()
```